sys/include/net/ng_sixlowpan/ctx.h

```diff
@@ -89,17 +103,28 @@ ng_sixlowpan_ctx_t *ng_sixlowpan_ctx_lookup_id(uint8_t id);
  * @param[in] id            The ID for the context.
  *                          Must be < @ref NG_SIXLOWPAN_CTX_SIZE.
  * @param[in] prefix        The prefix for the context.
- * @param[in] prefix_len    Length of @p prefix in bits. Must be > 0, when
- *                          @p ltime > 0.
- * @param[in] ltime         New lifetime of the context. The context will
- *                          be removed if 0.
+ * @param[in] prefix_len    Length of @p prefix in bits. Must be > 0.
+ * @param[in] ltime         New lifetime of the context. @p comp will be
+ *                          implicitly set to `false` if @p ltime == 0.
+ * @param[in] comp          Use for compression if true, do not use for
+ *                          compression, but still for decompression if false.
  *
  * @return  The new context on success.
  * @return  NULL, on error or on removal.
  */
 ng_sixlowpan_ctx_t *ng_sixlowpan_ctx_update(uint8_t id, const ng_ipv6_addr_t *prefix,
-                                            uint8_t prefix_len, uint16_t ltime);
+                                            uint8_t prefix_len, uint16_t ltime,
+                                            bool comp);
 
+/**
+ * @brief   Removes context.
+ *
+ * @param[in] id    A context ID.
+ */
+static inline void ng_sixlowpan_ctx_remove(uint8_t id)
+{
+    ng_sixlowpan_ctx_lookup_id(id)->prefix_len = 0;
```

Comment:
I would argue that one doesn't need a function for it, but that's only a weak opinion.